src/main/java/org/cbioportal/persistence/StudyViewRepository.java

```diff
@@ -1,9 +1,6 @@
 package org.cbioportal.persistence;
 
-import org.cbioportal.model.AlterationCountByGene;
-import org.cbioportal.model.ClinicalData;
-import org.cbioportal.model.ClinicalDataCount;
-import org.cbioportal.model.Sample;
+import org.cbioportal.model.*;
```

Comment:
i think IDE did this.  what's right?

Reply:
Do you need help with your ide settings?